src/util/get-prop.ts

```diff
@@ -0,0 +1,12 @@
+export const getProp = <T, V>(
```

Comment:
File name convention in TS is camelCase: https://jupiterone.atlassian.net/wiki/spaces/ENGINEERING/pages/13991937/TypeScript+Best+Practices#Use-camelCase-or-TitleCase-for-naming-files

Also, this is a non-optimal implementation of [Lodash.get](https://lodash.com/docs/#get). Any reason we shouldn't use that?

Reply:
I've updated the file name in my next PR. We chatted about `Lodash.get` in slack. My opinion is to use less dependencies and I'm open to any feedback about that idea.